src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,141 @@
+
+const IconWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: ${getFontSize('sm')};
+  width: 1rem;
+  height: 1rem;
+`;
+
+const BreadcrumbsWrapper = styled.div`
+  display: flex;
+`;
+
+// Build list of breadcrumbs interspersing a separator
+const insertSeparators = (items: React.ReactElement[]) => {
+  return items.reduce((prev, current, index) => {
+    let t = prev;
+    if (index < items.length - 1) {
+      t = t.concat(
+        current,
+        // eslint-disable-next-line react/no-array-index-key
+        <IconWrapper key={`separator-${index}`} aria-hidden>
+          <Icon
+            color={ColorTypes.neutral600}
+            name={SSCIconNames.chevronRight}
+          />
+        </IconWrapper>,
+      );
+    } else {
+      t.push(current);
+    }
+
+    return t;
+  }, []);
+};
+
+const Breadcrumbs: React.FC<BreadcrumbsProps> = ({ children, ...props }) => {
+  const itemsAfterCollapse = 2;
+  const itemsBeforeCollapse = 1;
+  const maxItems = 3;
+
+  // this renders the list of items only when the quantity
+  // is bigger than maxItems
+  const renderItemsBeforeAndAfter = (
+    allItems: React.ReactElement[],
+    allDropdownActions: ActionKinds<React.MouseEvent[]>[],
+  ) => {
+    const dropdown = () => {
+      return (
+        <DropdownMenu
+          key="breadcrumbs-dropdown"
+          actions={allDropdownActions}
+          paneWidth={270}
+          placement="bottom-start"
+          aria-hidden
+        >
+          <BreadcrumbItem>
+            <IconWrapper>
+              <Icon
+                aria-label="Open breadcrumbs menu"
+                name={SSCIconNames.ellipsisH}
+              />
+            </IconWrapper>
+          </BreadcrumbItem>
+        </DropdownMenu>
+      );
+    };
+
+    return [
+      ...slice(0, itemsBeforeCollapse, allItems),
+      dropdown(),
+      ...slice(allItems.length - itemsAfterCollapse, allItems.length, allItems),
+    ];
+  };
+
+  const childCount = React.Children.toArray(children).length;
+
+  const allItems = React.Children.map(children, (breadcrumbItem, index) => {
+    if (!React.isValidElement(breadcrumbItem)) {
+      return null;
+    }
+    return React.cloneElement(breadcrumbItem, {
+      isSelected: equals(index, childCount - 1),
+      ...props,
+    });
+  });
+
+  const allDropdownActions = reverse(
+    React.Children.map(children, (breadcrumbItem) => {
+      if (!React.isValidElement(breadcrumbItem)) {
+        return null;
+      }
+      return {
+        label: breadcrumbItem.props.children,
+        name: breadcrumbItem.props.children,
+        ...(isNotUndefined(breadcrumbItem.props.to) && {
+          to: breadcrumbItem.props.to,
+        }),
+        ...(isNotUndefined(breadcrumbItem.props.href) && {
+          href: breadcrumbItem.props.href,
+        }),
+      };
+    }),
+  );
+
+  return (
+    <BreadcrumbsWrapper>
+      <nav aria-label="Breadcrumb">
```

Comment:
Just a comment, this doesn't introduce any visual effect. Just for accessibility support: https://www.w3.org/TR/wai-aria-practices/examples/breadcrumb/index.html

 ```
 <nav aria-label="Breadcrumb" class="breadcrumb">
  <ol>
    <li>
      <a href="../../">
        WAI-ARIA Authoring Practices
      </a>
    </li>
    <li>
      <a href="../../#aria_ex">
        Design Patterns
      </a>
    </li>
    <li>
      <a href="../../#breadcrumb">
        Breadcrumb Pattern
      </a>
    </li>
    <li>
      <a href="./index.html" aria-current="page">
        Breadcrumb Example
      </a>
    </li>
  </ol>
</nav>
  ```